src/zenml/integrations/azure/orchestrators/azureml_orchestrator.py

```diff
@@ -0,0 +1,455 @@
+                step=step,
+                step_name=step_name,
+                env_name=deployment.pipeline_configuration.name,
+                image=image,
+                command=command,
+                arguments=arguments,
+            )
+
+        # Pipeline definition
+        pipeline_args = dict()
+        run_name = get_orchestrator_run_name(
+            pipeline_name=deployment.pipeline_configuration.name
+        )
+        pipeline_args["name"] = run_name
+
+        if compute_target := self._create_or_get_compute(ml_client, settings):
+            pipeline_args["compute"] = compute_target
+
+        @pipeline(force_rerun=True, **pipeline_args)  # type: ignore[call-overload, misc]
+        def azureml_pipeline() -> None:
+            """Create an AzureML pipeline."""
+            # Here we have to track the inputs and outputs so that we can bind
+            # the components to each other to execute them in a specific order.
+            component_outputs: Dict[str, Any] = {}
+            for component_name, component in components.items():
+                # Inputs
+                component_inputs = {}
+                if component.inputs:
+                    component_inputs.update(
+                        {i: component_outputs[i] for i in component.inputs}
+                    )
+
+                # Job
+                component_job = component(**component_inputs)
+
+                # Outputs
+                if component_job.outputs:
+                    component_outputs[component_name] = (
+                        component_job.outputs.completed
+                    )
+
+        # Create and execute the pipeline job
+        pipeline_job = azureml_pipeline()
+
+        if settings.mode == AzureMLComputeTypes.SERVERLESS:
+            pipeline_job.settings.default_compute = "serverless"
+
+        # Scheduling
+        if schedule := deployment.schedule:
+            try:
+                schedule_trigger: Optional[
+                    Union[CronTrigger, RecurrenceTrigger]
+                ] = None
+
+                start_time = None
+                if schedule.start_time is not None:
+                    start_time = schedule.start_time.isoformat()
+
+                end_time = None
+                if schedule.end_time is not None:
+                    end_time = schedule.end_time.isoformat()
+
+                if schedule.cron_expression:
+                    # If we are working with a cron expression
+                    schedule_trigger = CronTrigger(
+                        expression=schedule.cron_expression,
+                        start_time=start_time,
+                        end_time=end_time,
+                        time_zone=TimeZone.UTC,
+                    )
+
+                elif schedule.interval_second:
+                    # If we are working with intervals
+                    interval = schedule.interval_second.total_seconds()
```

Comment:
Should we maybe warn here that we can't use their exact configured seconds? Seems like if the interval is 1.5 days, it will still run every day for example?

Reply:
Or can we just use the `minute` frequency in all cases, to have the closest approximation?

Reply:
Not a bad idea. I switched to `minute`s and still added the warning if the number is not divisible by 60.